docker/src/main/java/brooklyn/entity/container/docker/DockerHostSshDriver.java

```diff
@@ -289,13 +289,25 @@ public String getVersion() {
     }
 
     private String installDockerOnUbuntu() {
-        String version = getVersion();
-        log.debug("Installing Docker version {} on Ubuntu", version);
+        final String dockerVersion = getVersion();
```

Comment:
It's good practice
"In general, declare all variables final if they are not modified after being initialized or set to some value."
https://docs.oracle.com/cd/E19159-01/819-3681/abebf/index.html